### PR TITLE
Preparation for release

### PR DESCRIPTION
**Pull request by gcor**
* add analytics
* add team secion
* add main action big button Го к нам
* increate ekb io logo size
* open tally forms by clicking action buttons

### CODE REVIEW

File: src/components/team/Team.module.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.team-grid {
```
Comment (gcor) [inline src/components/team/Team.module.css:1]: copypaste

File: src/pages/_document.tsx [hunk 2]
```diff
@@ -45,10 +45,38 @@ export default function Document() {
         <link rel="manifest" href="/site.webmanifest" />
 
         <link rel="dns-prefetch" href="https://mc.yandex.ru/" />
+
+        <script></script>
```
Comment (sashachabin) [inline src/pages/_document.tsx]: empty script?

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -45,10 +45,38 @@ export default function Document() {
...
       </Head>
       <body>
         <Main />
         <NextScript />
+
+        <div
+          dangerouslySetInnerHTML={{
+            __html: `<script>
```
Comment (sashachabin) [inline src/pages/_document.tsx]: Let's shorten `<script dangerouslySetInnerHTML={{ __html: ...`

File: src/pages/_document.tsx [hunk 4]
```diff
@@ -45,10 +45,38 @@ export default function Document() {
...
+                   trackLinks:true,
+                   accurateTrackBounce:true
+              });
+            </script>
+            `,
+          }}
+        />
+
+        <div
+          dangerouslySetInnerHTML={{
```
Comment (a-chabin) [inline src/pages/_document.tsx:75]: Let's shorten `<script dangerouslySetInnerHTML={{ __html: ...`

File: src/pages/_document.tsx [hunk 5]
```diff
@@ -45,10 +45,38 @@ export default function Document() {
...
+            __html: `<script async src="https://tally.so/widgets/embed.js"></script>
```
Comment (sashachabin) [inline src/pages/_document.tsx]: `<sript src>` not working? :(

File: src/components/team/TeamPerson.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+import { ITeamPerson } from "./types";
+
+const cx = classNames.bind(styles);
+
+function Person({ name, role, photo }: ITeamPerson) {
+  return (
+    <figure className={cx("team-person__inner")}>
+      {photo.length > 0 && (
+        <div className={cx("team-person__photo")}>
+          <Image src={photo} width={300} height={300} alt="" />
```
Comment (sashachabin) [inline src/components/team/TeamPerson.tsx]: No, it's better not to use this thing :(
We often go beyond the limits of Vercel.

File: next.config.js [hunk 7]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-}
+  images: {
+    domains: ["ekaterinburg.design"],
```
Comment (sashachabin) [inline next.config.js]: Let's add `images: { unoptimized: true }`. And maybe remove `domains: ["ekaterinburg.design"]`?

**Pull request merged**